themes/black-purple-2025/layouts/_partials/theme-toggle.html

```
@@ -0,0 +1,11 @@
+{{- /* Theme switcher UI: label + hidden checkbox + sliding sun/moon icons */ -}}
+<div>
+  <label class="inline-flex items-center cursor-pointer group" title="{{ i18n "toggle_theme_title" }}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)